From 4c68fcc7d7d8a92555376ab02e9edb6e4375b2d4 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey.doherty@intel.com>
Date: Tue, 1 Jan 2019 14:26:14 +0000
Subject: [linux-steam-integration 18/26] Update copyright ranges to be correct

Previous copyright assignment was to myself but the range doesn't account
for the project WIPs and lifetime, so retroactively fit them to the right
year range (project duration). Now set ongoing copyright for future and
current changes.

Signed-off-by: Ikey Doherty <ikey.doherty@intel.com>
---
 README.md                                | 2 +-
 src/common/files.c                       | 3 ++-
 src/common/log.c                         | 3 ++-
 src/common/vdf.c                         | 3 ++-
 src/frontend/main-window.c               | 3 ++-
 src/frontend/main.c                      | 3 ++-
 src/intercept/main.c                     | 3 ++-
 src/lsi/lsi.c                            | 3 ++-
 src/redirect/main.c                      | 3 ++-
 src/redirect/profile.c                   | 3 ++-
 src/redirect/profiles/ark.c              | 3 ++-
 src/redirect/profiles/project_highrise.c | 3 ++-
 src/redirect/unity.c                     | 3 ++-
 src/shim/lsi-exec.c                      | 3 ++-
 src/shim/main.c                          | 3 ++-
 src/shim/shim.c                          | 3 ++-
 16 files changed, 31 insertions(+), 16 deletions(-)

diff --git a/README.md b/README.md
index bdf97a8..26c9aa4 100644
--- a/README.md
+++ b/README.md
@@ -76,7 +76,7 @@ please check the [technical README document](https://github.com/clearlinux/linux
 
 Copyright © 2016-2018 Ikey Doherty
 
-Copyright © 2018 Intel Corporation
+Copyright © 2018-2019 Intel Corporation
 
 linux-steam-integration is available under the terms of the `LGPL-2.1` license.
 
diff --git a/src/common/files.c b/src/common/files.c
index c0dd887..b92ebb7 100644
--- a/src/common/files.c
+++ b/src/common/files.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/common/log.c b/src/common/log.c
index 2f69af7..9d8ee42 100644
--- a/src/common/log.c
+++ b/src/common/log.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/common/vdf.c b/src/common/vdf.c
index 26c4281..4675366 100644
--- a/src/common/vdf.c
+++ b/src/common/vdf.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/frontend/main-window.c b/src/frontend/main-window.c
index 75719f0..76baa8c 100644
--- a/src/frontend/main-window.c
+++ b/src/frontend/main-window.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/frontend/main.c b/src/frontend/main.c
index 88e4c55..9a25b9b 100644
--- a/src/frontend/main.c
+++ b/src/frontend/main.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/intercept/main.c b/src/intercept/main.c
index aa59a99..5fa77c5 100644
--- a/src/intercept/main.c
+++ b/src/intercept/main.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/lsi/lsi.c b/src/lsi/lsi.c
index 6efe6d2..c073262 100644
--- a/src/lsi/lsi.c
+++ b/src/lsi/lsi.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/redirect/main.c b/src/redirect/main.c
index 50d8d45..ea832fc 100644
--- a/src/redirect/main.c
+++ b/src/redirect/main.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/redirect/profile.c b/src/redirect/profile.c
index 5215380..cd8376c 100644
--- a/src/redirect/profile.c
+++ b/src/redirect/profile.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/redirect/profiles/ark.c b/src/redirect/profiles/ark.c
index 30847a7..9d50d00 100644
--- a/src/redirect/profiles/ark.c
+++ b/src/redirect/profiles/ark.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/redirect/profiles/project_highrise.c b/src/redirect/profiles/project_highrise.c
index 65d00b5..185a26c 100644
--- a/src/redirect/profiles/project_highrise.c
+++ b/src/redirect/profiles/project_highrise.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/redirect/unity.c b/src/redirect/unity.c
index 635f82a..2c279a8 100644
--- a/src/redirect/unity.c
+++ b/src/redirect/unity.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/shim/lsi-exec.c b/src/shim/lsi-exec.c
index 22f34d6..bf36d6b 100644
--- a/src/shim/lsi-exec.c
+++ b/src/shim/lsi-exec.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/shim/main.c b/src/shim/main.c
index e656def..5316631 100644
--- a/src/shim/main.c
+++ b/src/shim/main.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
diff --git a/src/shim/shim.c b/src/shim/shim.c
index 944e07e..a7ddce7 100644
--- a/src/shim/shim.c
+++ b/src/shim/shim.c
@@ -1,7 +1,8 @@
 /*
  * This file is part of linux-steam-integration.
  *
- * Copyright © 2016-2017 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2016-2018 Ikey Doherty <ikey@solus-project.com>
+ * Copyright © 2018-2019 Intel Corporation
  *
  * linux-steam-integration is free software; you can redistribute it and/or
  * modify it under the terms of the GNU Lesser General Public License as
-- 
2.20.0

